instance.tf

```diff
@@ -0,0 +1,42 @@
+locals {
+  mypublicip = sensitive(file("~/.ssh/mypubip.txt"))
+}
+resource "aws_instance" "public" {
```

Comment:
could you please, create the private instance.

Reply:
Sure, its just in class it was public so I made it that way.